pages/asanas/[id].js

```diff
@@ -0,0 +1,80 @@
+import { getAsanaByID, getAllAsanas } from "../../db";
+import Image from "next/image";
+import styled from "styled-components";
+
+export async function getStaticPaths() {
+  const asanas = getAllAsanas();
+  const ids = asanas.map((asana) => asana.id);
+  return {
+    paths: ids.map((id) => ({ params: { id: id } })),
+    fallback: false,
+  };
+}
+
+export async function getStaticProps(context) {
+  const { id } = context.params;
+  const asana = getAsanaByID(id);
+  return {
+    props: { asana: asana },
+  };
+}
+
+export default function Asana({ asana }) {
+  return (
+    <StyledArticle>
+      <Image
+        src={asana.img_url}
+        alt={asana.english_name}
+        width={230}
+        height={230}
+        layout="fixed"
+      />
+      <h2>{asana.english_name}</h2>
+      <h3>{asana.sanskrit_name}</h3>
+      <p>{asana.description}</p>
+
+      <StyledList>
+        {asana.level.map((item, index) => {
```

Comment:
One thing regarding your data structure: The property level on your asana objects should be called levels, since it is an array